src/clients/api/queries/getPendingRewards/formatOutput/formatRewardSummaryData.ts

```diff
@@ -21,7 +27,10 @@ export type RewardSummary = [
   ][],
 ];
 
-function formatRewardSummaryData(rewardSummary: RewardSummary): FormatRewardSummaryDataOutput {
+function formatRewardSummaryData(
+  rewardSummary: RewardSummary,
+  rewardTokenPrices: ReturnType<typeof formatTokenPrices>,
+): FormatRewardSummaryDataOutput {
```

Comment:
Nit: I'd use an object to define these params, so that it can be easily expanded later on if needed.